src/app_test.py

```diff
@@ -30,10 +56,24 @@ def test_get_dataset_schema(client):
     assert response.status_code == 200
 
 
-def test_get_dataset_schemas(client):
+def test_query_schemas(client, database):
+    get_schemas = database.get_schemas
```

Comment:
Hey Daniel, looking to add in a doc string for our unit tests as well, here is the example from the other team.

`    """
     What am I testing:
     AC-1.1 - The ability to submit a CI (wellformed) to the API endpoint, and the correct response is returned with the version.
     AC-1.3 - When a CI is published in the response the datetime field is present with an ISO8601 value. (2023-01-24T13:56:38Z)
     """`

Could we add those in? Happy to discuss as a ways of working?

Reply:
Yes, done.